test/server/hot_restart_impl_test.cc

```diff
@@ -50,8 +52,10 @@ class HotRestartImplTest : public testing::Test {
 
 TEST_F(HotRestartImplTest, versionString) {
   setup();
-  EXPECT_EQ(hot_restart_->version(), Envoy::Server::SharedMemory::version(
-                                         options_.maxStats(), options_.maxStatNameLength()));
+  EXPECT_EQ(hot_restart_->version(),
+            Envoy::Server::SharedMemory::version(options_.maxStats(),
+                                                 options_.maxObjNameLength() +
```

Comment:
I think this is correct, but as per @mattklein123's request, can you make sure the version string is the same before and after your change (as a manual check)?

Reply:
with the new build its `9.200.16384.127` and with master build, its `9.200.16384.127` . so it matches!